src/styles/global.css

```diff
@@ -29,6 +29,12 @@
       rgba(84, 132, 234, 0.2) 51.56%,
       rgba(58, 142, 137, 0.2) 100%
     );
+    --radial-gradient: radial-gradient(
```

Comment:
We should use a less generic name here

Reply:
Good call, do you know if we have a gradient from the DS that we could use in this case? will take a look and try to delete this one.